pkg/gardenlet/apis/config/v1alpha1/defaults_test.go

```diff
@@ -179,6 +180,36 @@ var _ = Describe("Defaults", func() {
 				Expect(obj.Logging.Loki.Enabled).To(PointTo(Equal(false)))
 				Expect(obj.Logging.Loki.Garden).NotTo(BeNil())
 				Expect(obj.Logging.Loki.Garden.Storage).To(PointTo(Equal(resource.MustParse("100Gi"))))
+				Expect(obj.Logging.ShootEventLogging).NotTo(BeNil())
+				Expect(obj.Logging.ShootEventLogging.Enabled).To(PointTo(Equal(false)))
+			})
+
+			It("should not overwrite custom settings", func() {
+				gardenLokiStorage := resource.MustParse("10Gi")
+				expectedLogging := &Logging{
+					Enabled: pointer.BoolPtr(true),
```

Comment:
```suggestion
					Enabled: pointer.Bool(true),
```

`<type>Ptr()` functions are only present for backwards compatibility.